engine/include/cubos/engine/gizmos/gizmos.hpp

```diff
@@ -0,0 +1,89 @@
+/// @file
+/// @brief Resource @ref cubos::engine::Gizmos.
+/// @ingroup gizmos-plugin
+
+#pragma once
+
+#include <vector>
+
+#include <glm/vec3.hpp>
+
+#include <cubos/core/gl/render_device.hpp>
+
+#include <cubos/engine/cubos.hpp>
+
+namespace cubos::engine
+{
+    class GizmosRenderer;
+
+    /// @brief Resource which queues for drawing gizmos, basic primitives useful for debugging and tools.
+    ///
```

Comment:
I think you should explain here in the docs of the class the semantics of having two gizmo objects with the same Id. IIRC two gizmos with the same ID should behave as a single 'object', right?

Reply:
gizmos with the same ID will share interaction functionality. There isn't any interaction yet, so i'm not sure how to document that.